localization/ekf_localizer/include/ekf_localizer/extended_kalman_filter_module.hpp

```diff
@@ -0,0 +1,96 @@
+// Copyright 2018-2019 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef EKF_LOCALIZER__EXTENDED_KALMAN_FILTER_MODULE_HPP_
+#define EKF_LOCALIZER__EXTENDED_KALMAN_FILTER_MODULE_HPP_
+
+#include "ekf_localizer/hyper_parameters.hpp"
+#include "ekf_localizer/state_index.hpp"
+#include "ekf_localizer/warning.hpp"
+
+#include <kalman_filter/kalman_filter.hpp>
+#include <kalman_filter/time_delay_kalman_filter.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <geometry_msgs/msg/pose_stamped.hpp>
+#include <geometry_msgs/msg/pose_with_covariance_stamped.hpp>
+#include <geometry_msgs/msg/transform_stamped.hpp>
+#include <geometry_msgs/msg/twist_stamped.hpp>
+#include <geometry_msgs/msg/twist_with_covariance_stamped.hpp>
+
+#include <memory>
+
+struct EKFDiagnosticInfo
```

Comment:
build error

> /__w/autoware.universe/autoware.universe/localization/ekf_localizer/include/ekf_localizer/ekf_localizer.hpp:51:8: error: redefinition of ‘struct EKFDiagnosticInfo’
>    51 | struct EKFDiagnosticInfo
>       |        ^~~~~~~~~~~~~~~~~
> In file included from /__w/autoware.universe/autoware.universe/localization/ekf_localizer/include/ekf_localizer/ekf_localizer.hpp:19,
>                  from /__w/autoware.universe/autoware.universe/localization/ekf_localizer/src/ekf_localizer.cpp:15:
> /__w/autoware.universe/autoware.universe/localization/ekf_localizer/include/ekf_localizer/extended_kalman_filter_module.hpp:34:8: note: previous definition of ‘struct EKFDiagnosticInfo’
>    34 | struct EKFDiagnosticInfo

Reply:
Thank you! Fixed: https://github.com/autowarefoundation/autoware.universe/pull/5424/commits/41da3836a33e4897a91ff1f6e213704f5a83a48f